packages/mask/src/components/CompositionDialog/Composition.tsx

```diff
@@ -51,16 +51,31 @@ export function Composition({ type = 'timeline', requireClipboardPermission }: P
     // #region Open
     const [open, setOpen] = useState(false)
     const [isOpenFromApplicationBoard, setIsOpenFromApplicationBoard] = useState(false)
+    const [recoverOverflow, setRecoverOverflow] = useState(false)
+
     const onClose = useCallback(() => {
         setOpen(false)
+        if (recoverOverflow) {
```

Comment:
All dialogs have an overflow problem. Let's patch this in the `<InjectedDialog />` component.